handlers/secrets.go

```diff
@@ -75,3 +81,86 @@ func makeSecretsArray(c *client.Client, secretNames []string) ([]*swarm.SecretRe
 
 	return values, nil
 }
+
+func secretGetHandler(c *client.Client, w http.ResponseWriter, r *http.Request) {
+	secrets, err := c.SecretList(context.Background(), types.SecretListOptions{})
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+
+		return
+	}
+
+	var results []requests.Secret
+
+	for _, s := range secrets {
+		results = append(results, requests.Secret{Name: s.Spec.Name})
+	}
+
+	bs, marshalErr := json.Marshal(results)
+
```

Comment:
The vertical spacing should be reduced a little bit by removing the empty lines between the err assignment and th error check, e.g. 

```golang
bs, marshalErr := json.Marshal(results)
if marshalErr != nil {
    w.WriteHeader(http.StatusInternalServerError)
    return
}
```

Reply:
Done